spec/interactors/users/check_credentials_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+
+describe Users::CheckCredentials do # rubocop:disable Metrics/BlockLength
```

Comment:
What are you check in this spec? You should have user in db and check that with right credentials it authenticate user and put him to context.